store/encode.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Tim Shannon. All rights reserved.
```

Comment:
This comes from badherhold, it could be included directly in `index.go` if that is preferable.